scripts/generate-unicode-data.js

```diff
@@ -0,0 +1,38 @@
+let regenerate = require('regenerate');
+let unicodeVersion = '8.0.0';
+
+let ascii = c => c < 128;
+let nonAscii = c => c >= 128;
+
+let makeCharRegex = s => `/^${regenerate().add(...s).toString()}$/`;
+let makeCharArray = s => JSON.stringify(regenerate().add(...s).toArray());
+
+let whitespace = [0x09, 0x0B, 0x0c, 0x20, 0xA0, 0xFEFF].concat(require(`unicode-${unicodeVersion}/General_Category/Space_Separator/code-points`));
```

Comment:
Eh... the idea was that this is all of the whitespace. But sure, I can remove them.